octoprint_octorelay/__init__.py

```diff
@@ -153,7 +154,8 @@ def on_api_command(self, command, data):
         if command == GET_STATUS_COMMAND: # API command to get relay status
             return self.handle_get_status_command(data["pin"])
         if command == UPDATE_COMMAND: # API command to toggle the relay
-            return self.handle_update_command(data["pin"])
+            target = data.get("target", "").upper() # optional parameter for target state
+            return self.handle_update_command(data["pin"],target)
```

Comment:
Consider operating booleans within JSON payload.

```suggestion
            target = data.get("target") # optional parameter for target state
            return self.handle_update_command(data["pin"], None if target is None else bool(target))
```

- `get` method will return the specified value or `None`: https://www.programiz.com/python-programming/methods/dictionary/get
- The statement `None if target is None else bool(target)` will ensure the type expected by `handle_update_command` (Optional[bool])

Reply:
fixed in feed05c